src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -1040,6 +1040,18 @@ protected static GenotypesContext updateGenotypesWithMappedAlleles(final Genotyp
      * <DEL>/C => <DEL>/C
      *
      * @param alleles a non-null list of alleles that we want to clip
+     * @param isReverse if true, clip from the end of the allele if false, clip from the beginning of the allele.
+     * When isReverse is true:
+     * AC/A => AC/A
+     * AT/AT => A/A
+     * CCG/CG => CC/C
+     * TGCC/TGCC => T/T
+     * When isReverse is false:
```

Comment:
Could you add an example with three alleles eg TTC/GTC/GTA -> no change?  And do the same for forward clipping.